src/main.tsx

```diff
@@ -1,14 +1,43 @@
+import {
+  ColorSchemeProvider,
+  createEmotionCache,
+  MantineProvider,
+} from '@mantine/core';
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import rtlPlugin from 'stylis-plugin-rtl';
 
 import { App } from './App';
 
+const rtlCache = createEmotionCache({
+  key: 'mantine-rtl',
+  stylisPlugins: [rtlPlugin],
+});
+
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement,
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <ColorSchemeProvider
+      colorScheme="light"
+      toggleColorScheme={() => {
+        console.log('');
+      }}
+    >
+      <MantineProvider
+        withGlobalStyles
+        withCSSVariables
+        emotionCache={rtlCache}
+        theme={{
+          dir: 'rtl',
+          fontFamily: 'Iranian Sans, sans-serif !important',
+          colorScheme: 'light',
```

Comment:
I think it's a better idea to have separate file for this
and why is `!important` there